account_fiscal_position_partner_type/views/res_partner.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <record id="view_partner_form_fiscal_position_type" model="ir.ui.view">
+        <field name="name">view.partner.form.fiscal.position.type</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='vat']" position="before">
+                <field
+                    name="fiscal_position_type"
+                    attrs="{'invisible': [('parent_id', '=', True)]}"
```

Comment:
We should put this together the fiscal position, not the VAT.

Reply:
I thought that, but if it is a field that will have to be configured every time a partner is created that is next to the 'VAT' field, I see it more correct, since it can be configured faster.

Reply:
OK, you are right. The fiscal position will be auto-applied, so no configuration is done there.